ivy.el

```diff
@@ -200,6 +200,12 @@ See https://github.com/abo-abo/swiper/wiki/ivy-display-function."
   '((ivy-completion-in-region . ivy-display-function-overlay))
   "An alist for customizing `ivy-display-function'.")
 
+(defvar ivy-display-schemes
+  '((ivy-overlay
+     :display ivy-display-function-overlay
+     :cleanup ivy-overlay-cleanup))
+  "An alist of display schemes.")
```

Comment:
How about providing a bit more information with something like "Map display schemes to their property list."